src/ant/doc/README.md

```diff
@@ -0,0 +1,39 @@
+# Antenna Rule Checker
+
+This tool checks antenna violations based on an input LEF and DEF file, a report
```

Comment:
'based on input LEF and DEF'? not really. it uses opendb. this is pointless in the context of an openroad command

Reply:
I rephrased it, but since it is fetching the factors from the def/lef, I thought I should point that out.

Reply:
If I start with a verilog file and never read a DEF file (see test/flow.tcl) it is not based on a DEF file.
It is pointless to even mention LEF/DEF. OR commands all work on the database, not LEF/DEF, and it would
not make sense for every command documentation to say they are "based on LEF/DEF".

Reply:
makes sense, I removed that part